src/UIComponents/SaveModel.jsx

```diff
@@ -41,64 +57,96 @@ class SaveModel extends Component {
     fields.push({
       id: "potentialMisuses",
       text: "How can this model be potentially misused?",
+      description: "Consider whether this model was trained on data that can identify subgroups, whether the data has adequate representation of subgroups, and whether this data could be used to inform decisions central to human life.",
       placeholder: "Write a brief description."
     });
-    fields.push({
-      id: "identifySubgroup",
-      type: "checkbox",
-      text: "Has this model been trained on data that can identify a subgroup?"
-    });
-    fields.push({
-      id: "representSubgroup",
-      type: "checkbox",
-      text: "Have we ensured the data has adequate representation of subgroups?"
-    });
-    fields.push({
-      id: "decisionsLife",
-      type: "checkbox",
-      text:
-        "Could this model be used to inform decisions central to human life?"
-    });
 
     return fields;
   };
 
   render() {
+    const nameField = {
+      id: "name",
+      text: "What will you name the model? (required)"
+    };
+
+    const arrowIcon = this.state.showColumnDescriptions
+      ? 'fa fa-caret-up' : 'fa fa-caret-down';
+
+    const columnCount = this.getColumnFields().length;
+
     return (
       <div style={styles.panel}>
         <div style={styles.largeText}>Model Details</div>
         <div style={styles.scrollableContentsTinted}>
           <div style={styles.scrollingContents}>
-            {this.getFields().map(field => {
-              return (
-                <div key={field.id} style={styles.cardRow}>
-                  {field.type === "checkbox" && (
-                    <input
-                      type="checkbox"
-                      onChange={event => this.handleChange(event, field.id)}
-                    />
-                  )}
-
-                  <label>{field.text}</label>
-
-                  {field.type !== "checkbox" && !field.answer && (
-                    <div>
-                      <textarea
-                        rows="2"
-                        onChange={event =>
-                          this.handleChange(event, field.id, field.isColumn)
-                        }
-                        placeholder={field.placeholder}
-                      />
-                    </div>
-                  )}
-
-                  {field.type !== "checkbox" && field.answer && (
-                    <div>{field.answer}</div>
-                  )}
+            <div key={nameField.id} style={styles.cardRow}>
+              <label>{nameField.text}</label>
+              <div>
+                <textarea
+                  rows="1"
+                  onChange={event =>
+                    this.handleChange(event, nameField.id, nameField.isColumn)
+                  }
+                />
```

Comment:
The name field still seems to be vertically expandable and supports multiple rows of text.  Can we get it to be a single line only?